### PR TITLE
chore(kinds): merge into mono repo

### PR DESCRIPTION
**Pull request by emosbaugh**
#### What this PR does / why we need it:

Merges kinds repo into ec

https://github.com/replicatedhq/airgap/pull/189
https://github.com/replicatedhq/kots/pull/4854

#### Does this PR require a release note?

```release-note

```

### CODE REVIEW

File: kinds/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+This allows us to use the client-go and other functionality to parse and ensure conformance.
+
+## Publishing kinds
+
+To publish a new version of the github.com/replicatedhq/embedded-cluster/kinds package, tag and push to with the format `kinds/v1.0.0` with a leading `v`.
+This is a Go feature. See the docs on [Multi-Module Repositories](https://go.dev/wiki/Modules#faqs--multi-module-repositories) for more information.
+
+```bash
+git tag -a kinds/v1.0.0 -m "Release v1.0.0"
+git push origin kinds/v1.0.0
```
Comment (emosbaugh) [inline kinds/README.md:14]: FYI

File: .github/workflows/kinds.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: kinds/go.mod
+          cache-dependency-path: kinds/*.sum
+
+      - name: Unit tests
+        run: make -C kinds unit-tests
+
+  validate-success:
```
Comment (emosbaugh) [inline .github/workflows/kinds.yaml:57]: TODO: add branch protection

**Comment by laverya** [comment]
If we merge embedded-cluster-kinds into EC, will kots be able to import things properly still?

File: .github/workflows/kinds.yaml (hunk 2 above)
Comment (laverya) [reply .github/workflows/kinds.yaml:57]: this might need to be a different name, not sure (current 'ci' workflow also has a 'validate-success' job)

Comment (emosbaugh) [reply .github/workflows/kinds.yaml:57]: Ill follow up with another pr if it does

**Comment by emosbaugh** [comment]
> If we merge embedded-cluster-kinds into EC, will kots be able to import things properly still?

@laverya It seems to work
https://github.com/replicatedhq/kots/pull/4854

**Review by sgalsaleh — APPROVED** [review]

**Pull request merged**